tests/unit/test_opentelemetry_tracing.py

```diff
@@ -0,0 +1,101 @@
+import sys
+import pytest
+
+import mock
+
+from google.cloud.bigquery import opentelemetry_tracing
+
+try:
+    import opentelemetry
+    from opentelemetry import trace
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
+    from opentelemetry.sdk.trace.export.in_memory_span_exporter import (
+        InMemorySpanExporter,
+    )
+except ImportError:
+    opentelemetry = None
+
+from six.moves import reload_module
+
+TEST_SPAN_NAME = "bar"
+TEST_SPAN_ATTRIBUTES = {"foo": "baz"}
+
+
+@pytest.mark.skipif(opentelemetry is None, reason="Require `opentelemetry`")
+@pytest.fixture
+def setup():
+    tracer_provider = TracerProvider()
+    memory_exporter = InMemorySpanExporter()
+    span_processor = SimpleExportSpanProcessor(memory_exporter)
+    tracer_provider.add_span_processor(span_processor)
+    trace.set_tracer_provider(tracer_provider)
+    yield memory_exporter
+
+
+@pytest.mark.skipif(opentelemetry is None, reason="Require `opentelemetry`")
```

Comment:
Not really familiar with pytest, but is opentelemetry really required for a test that checks that things work if opentelemetry is not installed?

Reply:
I believe there was an issue with this when Python 2.7 was run against all the tests in the pipeline. But I'll double check and remove it if we can run it against Python 2.7.

Reply:
Can confirm there is an issue with python 2.7 in this test.